kfs-cam/src/main/java/edu/arizona/kfs/module/cam/CamsConstants.java

```diff
@@ -14,4 +14,8 @@
 		public static final String ASSET_INVENTORY_UNIT_NULL ="AssetInventoryUnit Null";
 		
 	}
+	
+    public static class AssetCreateAndUpdateNotePrefixes {
```

Comment:
Is this new static class necessary? Using Intellij's ctrl-g "reference-finding" function, I don't see anything either referencing this class or the class constant; am I mistaken about this not being used?